src/api/vipps-login/index.ts

```diff
@@ -13,14 +13,14 @@ export default (server: Hapi.Server) => () => {
         projectId: PROJECT_ID
     })
 
-    const getClient = async () => {
+    const getClient = async (callbackUrl: string) => {
         const iss = await Issuer.discover(VIPPS_BASE_URL + '/access-management-1.0/access/');
```

Comment:
While we're here, maybe we should cache the `Issuer` in an outer scope and return the cached copy if has a value? Waiting for OIDC discovery for each authorization request is wasteful. The JWKS will be kept and automatically refreshed based on the expiry in the response.

Reply:
how about now?